src/main/java/uk/gov/hmcts/reform/sscs/helper/HearingsMapping.java

```diff
@@ -0,0 +1,21 @@
+package uk.gov.hmcts.reform.sscs.helper;
+
+import uk.gov.hmcts.reform.sscs.model.HearingWrapper;
+import uk.gov.hmcts.reform.sscs.model.single.hearing.HearingDeleteRequestPayload;
+
+public class HearingsMapping {
+
+    private static String CANCEL_REASON_TEMP = "AWAITING_LISTING";
+
+    private HearingsMapping() {
+
+    }
+
+    public static HearingDeleteRequestPayload buildDeleteHearingPayload(HearingWrapper wrapper) {
```

Comment:
Your test is only passing because CANCEL_REASON_TEMP and CANCELLATION_REASON  happen to resolve to the same string. You should probably pass in the Cancellation Reason code as an argument

Reply:
This test is no longer valid due to your other comment as we are not returning a cancelation reason anymore